elasticsearch-centos-3node/azuredeploy.json

```diff
@@ -0,0 +1,381 @@
+{
+  "$schema": "https://schema.management.azure.com/schemas/2015-01-01/deploymentTemplate.json#",
+  "contentVersion": "1.0.0.0",
+  "parameters": {
+    "adminUsername": {
+      "type": "string",
+      "metadata": {
+        "description": "Admin username"
+      }
+    },
+    "adminPassword": {
+      "type": "securestring",
+      "metadata": {
+        "description": "Admin password"
+      }
+    },
+	  "location": {
+      "type": "string",
+      "metadata": {
+        "description": "Location where resources will be provisioned"
+      }
+    },
+    "vmPrefix": {
+      "type": "string",
+      "metadata": {
+        "description": "Prefix to use for VM names"
+      }
+    },
+    "commonPrefix": {
+      "type": "string",
+      "metadata": {
+        "description": "Prefix to use for common resources"
+      }
+    }
+  },
+  "variables": {
+    "vmSize": "Standard_D1",
+    "clusterName": "elasticcluster",
+    "logTableName": "elasticscriptlog",
+    "imagePublisher" : "OpenLogic",
+    "imageOffer" : "CentOS",
+    "imageSku" : "7.1",
+    "availabilitySetName": "[concat(parameters('VMPrefix'), 'AvSet')]",
+    "storageName" : "[toLower(concat(parameters('commonPrefix'), uniqueString(resourceGroup().id)))]",
+    "storageAccountType": "Standard_LRS",
+    "dataDiskSize": "1023",
+    "virtualNetworkName":  "[concat(parameters('commonPrefix'), 'VNet')]" ,
+    "publicIPAddressName": "JumpboxIP",
+    "publicIPAddressType": "Dynamic",
+    "JBNSGName": "JumpboxNSG",
+    "subnetName": "elasticSubnet",
+	  "subnetAdmin": "adminSubnet",
+    "networkInterfaceName": "NIC",
+    "vnetID": "[resourceId('Microsoft.Network/virtualNetworks',variables('virtualNetworkName'))]",
+    "subnetRef": "[concat(variables('vnetID'),'/subnets/',variables ('subnetName'))]",
+	  "subnetAdminRef": "[concat(variables('vnetID'),'/subnets/',variables ('subnetAdmin'))]",
+    "numberOfNodes": 3,
+    "scriptBaseUrl": "https://raw.githubusercontent.com/Azure/azure-quickstart-templates/master/elasticsearch-centos-3node/",
+    "elasticScript": "elasticinstall.py",
+    "jumpboxScript" : "prep-jumpbox.ps1",
+    "installScripts": [
+        "[concat(variables('scriptBaseUrl'), variables('elasticScript'))]",
+        "[concat(variables('scriptBaseUrl'), 'sdc.layout')]"
+    ],
+    "jumpboxScripts": [
+        "[concat(variables('scriptBaseUrl'), variables('jumpboxScript'))]",
+        "[concat(variables('scriptBaseUrl'), 'get-started.ps1')]"
+    ]
+    
+  },
+  "resources": [
+    {
+      "apiVersion": "2015-05-01-preview",
```

Comment:
I suggest making the api version for all resources a variable and using the GA versions 2015-06-15